### DISCUSSION TITLE
Subtitle selector not working issue

### DISCUSSION THREAD

**Discussion by PsychonicJoe** [mo3rfan/syncplayer#23] [General]
Hello, it's me from the Playstore review asking about subtitle selector.
The issue is that the video I'm trying to play already has soft-coded subtitles, when I open the CC menu to load subtitles, no matter what the subtitles don't change, they just keep playing the default ones that are in the video. I tried an empty subtitles file and subtitles in another language but they don't change.
Also suddenly for some reason I can't load videos anymore, when I add a file the screen just keeps saying "press + to load a media file..." even though I did, and you can see that I have a video loaded from the PC clients but can't play anything on the phone.
(Edit: As for the video loading issue, some videos work but others don't)
I'm using a Lenovo Vibe k5note phone with Android 6.0
Thank you for your time and this amazing app.

**Comment by mo3rfan** [comment]
Thank you for the details! This is very helpful and I'll work on it.

**Reply by sidvud98** [reply]
Has the subtitle issue been resolved yet? Please do so, it is really necessary during this time of lockdown.

**Reply by PsychonicJoe** [reply]
Yeah I've been waiting too, hopefully it'll be fixed soon.
